modules/core/src/main/java/org/apache/synapse/config/xml/SynapseXMLConfigurationFactory.java

```diff
@@ -99,11 +100,11 @@ public SynapseConfiguration getConfiguration(OMElement definitions, Properties p
                     defineEventSource(config, elt, properties);
                 } else if (XMLConfigConstants.EXECUTOR_ELT.equals(elt.getQName())) {
                     defineExecutor(config, elt, properties);
-                } else if(XMLConfigConstants.MESSAGE_STORE_ELT.equals(elt.getQName())) {
+                } else if (XMLConfigConstants.MESSAGE_STORE_ELT.equals(elt.getQName())) {
                     defineMessageStore(config, elt, properties);
```

Comment:
It's better if we do not include unrelated code changes like formatting etc. to this PR.